### ISSUE TITLE
Dropbox screenshot failure - error unknown

### ISSUE THREAD

**Issue by joshuawagner** [vookimedlo/alfred-various-screenshots#8]
Hi and thanks for making such a great plugin! I'm trying to get this setup in Alfred 4. The only non-standard things I've done are to change the syntax for the filename. I did add the access token from a Dropbox 'app' that I created. Normal screenshots work fine but when I try to hold option to send it to Dropbox I get a 'Dropbox screenshot failure - error unknown' message. I click show on the popup but it doesn't take me anywhere. Any idea what I've done wrong or how to diagnose?

**Comment by vookimedlo** [comment]
Hi Joshua,

Does the dropbox works for you if you just download this workflow without any changes? Of course, the Dropbox token must be used.

I tried the Dropbox feature and it works for me.

Thanks,

Michal

**Comment by joshuawagner** [comment]
No, I deleted the workflow and downloaded the latest release from github. I
pasted in my access token and changed nothing else. I imagine it's
something on my end, maybe on the dropbox side? Is there a log file that
will indicate what error it's throwing? Any thoughts about what permissions
or customization is needed on the Dropbox side?
Thanks for your help,
Josh

**Comment by vookimedlo** [comment]
Joshua,
sorry for delay. Try to open the alfred preferences, select the workflow and enable the 'debugging mode'. It's the icon depicting a bug in the upper right corner.

Try to use the workflow and post the debug output here.

![image](https://user-images.githubusercontent.com/2312301/103813885-13d97400-5061-11eb-8c6a-0d5703e8aab5.png)

**Comment by thorstenmerl** [comment]
Same problem here... see the screenshot for more information.
![screenshot_2021-07-08_10-31-29](https://user-images.githubusercontent.com/71291080/124889880-cb7f1c80-dfd7-11eb-828c-fcee65eea589.png)
would be nice if you could provide some help; thanks in advance (and also thx for this workflow, that I use all the time)

**Comment by LampPrinter** [comment]
Same problem. Not working for me either.

**Comment by augusl** [comment]
![62AzXw3.png](https://i.imgur.com/62AzXw3.png)
The solution is: first give the file and folder read and write permissions, and then generate the token

**Comment by LampPrinter** [comment]
Still not working for me. @augusl

**Comment by joshuawagner** [comment]
> Joshua, sorry for delay. Try to open the alfred preferences, select the workflow and enable the 'debugging mode'. It's the icon depicting a bug in the upper right corner.
> 
> Try to use the workflow and post the debug output here.
> 
> ![image](https://user-images.githubusercontent.com/2312301/103813885-13d97400-5061-11eb-8c6a-0d5703e8aab5.png)

I don't know why, but I generated a new access token and now it works!
